### ISSUE TITLE
Can't control speaker with model fine tuned with cycle loss.

### ISSUE THREAD

**Issue by LorenzoBrugioni** [DigitalPhonetics/IMS-Toucan#97]
Good evening, I'm trying to do some fine tuning runs and I've obtained some checkpoints.
When I try in inference the model with run_interactive_demo.py, if I choose the model loaded from a checkpoint that hasn't undergone the cycle loss (phase2) everything's fine and I can control the speaker voice with  PATH_TO_REFERENCE_SPEAKER.
Whenever I load a checkpoint resulting from a phase1+phase2 fine tuning, I'm no longer able to choose the speaker with PATH_TO_REFERENCE_SPEAKER. 
For a specific language it just uses a voice present in the dataset with that language.

**Comment by Flux9665** [comment]
Interesting, maybe the cycle loss causes overfitting during finetuning. I usually finetune single-speaker models from multispeaker ones, I never tried finetuning a multi-speaker model from another multi-speaker model. I'll try it out and see if I can reproduce the problem and find an easy solution.

**Comment by LorenzoBrugioni** [comment]
Hi @Flux9665 , thank you for the answer.
I don't fully understand what you mean by finetuning single-speaker models, because by looking into the finetuning_example.py, I see that there are 4 datasets (2 english and 2 german), so I assumed there are 2 languages and 4 different speakers, and the fine tuning procedure on those would result in a multi-speaker multi-lingual model capable of speaking well those 2 languages and with the 4 different voices (in both languages).  Is it not intended like that?

Can you please explain what is your usual procedure, what is your intent when fine tuning and what the resulting model is capable of?

Thank you again and have a great day.
